src/chartmuseum/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: Host your own Helm Chart Repository
 name: chartmuseum
-version: 3.1.0
+version: 3.1.1
```

Comment:
Needs a version bump. 3.1.1 was released several days ago

Reply:
yes, +1 then OK to merge. Should be `3.2.0`